### ISSUE TITLE
Documentation is so hard to read and confusing about 'Templates'

### ISSUE THREAD

**Issue by ljakimagia** [logstash-plugins/logstash-output-elasticsearch#804]
Hello everyone, 

I find the documentation about '[Template](https://www.elastic.co/guide/en/logstash/6.4/plugins-outputs-elasticsearch.html#plugins-outputs-elasticsearch-template)' (*template*, *template_name*, *template_overwrite*, *manage_template*) really hard to read and confusing. And it's been some months now that I'm using the templates (note that I have running indexes with templates successfully in my projects). At first, I really thought I will need more experience with Template but today, I realize it's not the case: each time I'm going back to read the docs about Templates, I...really understand nothing. I find this StackOverflow thread today where a user explains very clearly each parameter :

```
Basically, manage_template toggles on/off the template management. template gives the path to the template in case template management is toggle on. template_name is the name under which the template is saved in ES. and template_overwrite tells Logstash whether to overwrite the template if one exists already under the same name. – Val Apr 13 at 5:18
```
https://stackoverflow.com/questions/49720821/how-to-set-an-elasticsearch-output-template-in-logstash 

Therefore (if I'm not the only one) it could be nice to update the docs accordingly.

Thanks a lot.
